src/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: min-content;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 16px;
+}
+
+.card__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 16px 3px 0;
+}
+
+.card__title {
+  font-weight: 500;
+  line-height: 18px;
+
+  width: 166px;
```

Comment:
Why have you hardcoded title width? It's probably should take all free space

Reply:
Okay, I got it, you haven't set width to `card`
Instead of setting width for every element, better to set width to parent, and then if there will be need to make block wider - set width: 100% (because due to design there is no options where card can be wider)

Because it's looks strange now, we know that card should have 200px width, but instead of setting it we've calculated how many width should content have and set it to elements

It's great to let content dictate container size instead of hardcoding it, but in this case we have hardcoded it twice 🙂  So let's make it a bit simpler